SingularityUI/app/models/TaskHistoryItem.coffee

```diff
@@ -6,7 +6,19 @@ class TaskHistoryItem extends Model
 
     ignoreAttributes: ['id', 'canBeRunNow']
 
-    initialize: ({ @taskId, @updatedAt, @lastTaskState }) ->
+    url: () ->
+        if @requestId and @runId
+            "#{ config.apiRoot }/history/request/#{ @requestId }/run/#{ @runId }"
+        else
+            # Currently the above URL is the ONLY place to fetch this model.
+            # If you don't have access to request ID and run ID use the TaskHistory
```

Comment:
once we have the taskId, we can also fetch this using that. I know the main use case with be by runId, but just wanted to clarify